nucypher/utilities/prometheus/collector.py

```diff
@@ -173,45 +171,18 @@ def __init__(self, domain: str, staker_address: ChecksumAddress, contract_regist
 
     def initialize(self, metrics_prefix: str, registry: CollectorRegistry) -> None:
         self.metrics = {
-            "current_period_gauge": Gauge(f'{metrics_prefix}_current_period', 'Current period', registry=registry),
-            "eth_balance_gauge": Gauge(f'{metrics_prefix}_staker_eth_balance', 'Ethereum balance', registry=registry),
             "token_balance_gauge": Gauge(f'{metrics_prefix}_staker_token_balance', 'NuNit balance', registry=registry),
-            "substakes_count_gauge": Gauge(f'{metrics_prefix}_substakes_count', 'Substakes count', registry=registry),
             "active_stake_gauge": Gauge(f'{metrics_prefix}_active_stake', 'Active stake', registry=registry),
-            "unlocked_tokens_gauge": Gauge(f'{metrics_prefix}_unlocked_tokens',
-                                           'Amount of unlocked tokens',
-                                           registry=registry),
-            "owned_tokens_gauge": Gauge(f'{metrics_prefix}_owned_tokens',
-                                        'All tokens that belong to the staker, including '
-                                        'locked, unlocked and rewards',
-                                        registry=registry),
         }
 
     def _collect_internal(self) -> None:
-        staking_agent = ContractAgency.get_agent(StakingEscrowAgent, registry=self.contract_registry)
-
-        # current period
-        self.metrics["current_period_gauge"].set(staking_agent.get_current_period())
-
         # balances
         nucypher_token_actor = NucypherTokenActor(registry=self.contract_registry,
                                                   domain=self.domain,
                                                   checksum_address=self.staker_address)
         self.metrics["eth_balance_gauge"].set(nucypher_token_actor.eth_balance)
         self.metrics["token_balance_gauge"].set(int(nucypher_token_actor.token_balance))
```

Comment:
It's unclear to me that this is relevant information - this gauge can probably be removed.